spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/builder/JaxRsClientMethodBuilderSpec.groovy

```diff
@@ -1008,6 +1008,7 @@ class JaxRsClientMethodBuilderSpec extends Specification implements WireMockStub
 					body([
 							alpha: $(anyAlphaUnicode()),
 							number: $(anyNumber()),
+							double: $(anyDouble()),
```

Comment:
shouldn't this be in quotes `"double"` since it's a keyword?

Reply:
worked without the quotes, but i have changed it to `aDouble` (as its already using `aBoolean`) to avoid confusion